yarn-project/prover-client/src/tx-prover/tx-prover.ts

```diff
@@ -7,36 +7,53 @@ import { type WorldStateSynchronizer } from '@aztec/world-state';
 import { type ProverConfig } from '../config.js';
 import { type VerificationKeys, getVerificationKeys } from '../mocks/verification_keys.js';
 import { ProvingOrchestrator } from '../orchestrator/orchestrator.js';
+import { CircuitProverAgent } from '../prover-pool/circuit-prover-agent.js';
+import { MemoryProvingQueue } from '../prover-pool/memory-proving-queue.js';
+import { type ProvingAgent } from '../prover-pool/prover-agent.js';
+import { type ProvingQueue } from '../prover-pool/proving-queue.js';
 import { TestCircuitProver } from '../prover/test_circuit_prover.js';
 
 /**
  * A prover accepting individual transaction requests
  */
 export class TxProver implements ProverClient {
   private orchestrator: ProvingOrchestrator;
+  private queue: ProvingQueue;
```

Comment:
Shouldn't this use `ProverPool`?

Reply:
Yes, I had just added `ProverPool` this morning to help with the unit tests and forgot to update all usages.